pkg/ottl/ottlfuncs/func_fnv_test.go

```diff
@@ -0,0 +1,50 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package ottlfuncs
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/open-telemetry/opentelemetry-collector-contrib/pkg/ottl"
+)
+
+func Test_FNV(t *testing.T) {
```

Comment:
Please add a test for the `.Get` returning an error.  You can use https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/04b584845b83903895e735f0f3286f5afefaabf6/pkg/ottl/ottlfuncs/func_convert_case_test.go#L214 as an example